pandas/io/formats/style.py

```diff
@@ -214,6 +214,8 @@ def _repr_html_(self) -> str:
         """
```

Comment:
IIRC aren't we supposed to have a ```_repr_latex_``` ?

Reply:
I don't know. See if you prefer the new pattern, simple change.